tests/VarExporterTest.php

```diff
@@ -151,6 +151,42 @@ public function testInlineNumericScalarArray()
         $this->assertExportEquals($expected, $var, VarExporter::INLINE_NUMERIC_SCALAR_ARRAY);
     }
 
+    public function testExportDateTime()
+    {
+        $timezone = new \DateTimeZone('Europe/Berlin');
+        $format = 'Y-m-d H:i:s.u';
+
+        $var = \DateTime::createFromFormat($format, '2020-03-09 18:51:23.000000', $timezone);
+
+        $expected = <<<'PHP'
+\DateTime::__set_state([
+    'date' => '2020-03-09 18:51:23.000000',
+    'timezone_type' => 3,
+    'timezone' => 'Europe/Berlin'
+])
+PHP;
+
+        $this->assertExportEquals($expected, $var);
+    }
+
+    public function testDateTimeZoneSetState()
```

Comment:
Shouldn't this be called `testExportDateTimeImmutable`?

Reply:
Oh, yes, you are right.. I tested the `DateTimeZone` in that method and forgot to rename it afterwards. I'll correct that later today.